sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLCommandTestUtils.scala

```diff
@@ -108,4 +108,15 @@ trait DDLCommandTestUtils extends SQLTestUtils {
     }
     size
   }
+
+  def cacheRelation(name: String): Unit = {
+    assert(!spark.catalog.isCached(name))
```

Comment:
Just a question. Why do we assert this?

Reply:
To be sure that `CACHE TABLE` really works. If a table was already cached, the second assert after `sql(s"CACHE TABLE $name")` is useless.